bcda/bcdacli/cli.go

```diff
@@ -359,10 +359,49 @@ func setUpApp() *cli.App {
 					Usage:       "Directory where CCLF files are located",
 					Destination: &filePath,
 				},
+				cli.StringFlag{
+					Name:        "filesource",
+					Usage:       "Source of files. Must be one of 'local', 's3'. Defaults to 'local'",
```

Comment:
This is more philosophical, since we don't run most of these commands manually, but rather through say jenkins - the below doesn't appear to enforce only local or s3 (rather anything that's not s3 is treated as local), do we want an extra level of nuance for the cli commands? Eg requiring the filesource to be specified, directory must be specified, rather than letting cli fail if not provided?

Reply:
(My personal thought is yes, but if we choose to, it can be a separate ticket)

Reply:
Since this is a new parameter, keeping a default of `local` allows us to transition smoothly (the Jenkins jobs don't specify filesource right now, so it's a bit of a chicken-or-egg situation.)

But I can see it would be helpful to add the enforcement in a followup ticket for both CCLF and opt out ingestion.